src/main/java/tool/DateTime.java

```diff
@@ -0,0 +1,38 @@
+package tool;
+
+public class DateTime {
```

Comment:
Java has a standard `LocalDateTime` class, in `java.time`. It might be easier to work with the existing libraries instead of writing your own.